tests/integration/backward_compatible/proxy/vector_collection_test.py

```diff
@@ -0,0 +1,151 @@
+import os
+import unittest
+
+import pytest
+
+from tests.base import SingleMemberTestCase
+from tests.util import random_string, compare_client_version
+
+try:
+    from hazelcast.vector import IndexConfig, Metric, Document, Vector, Type
+except ImportError:
+    # backward compatibility
+    pass
+
+
+@unittest.skipIf(
+    compare_client_version("5.5") < 0, "Tests the features added in 5.5 version of the client"
+)
+@pytest.mark.enterprise
+class VectorCollectionTest(SingleMemberTestCase):
+    @classmethod
+    def configure_cluster(cls):
+        path = os.path.abspath(__file__)
+        dir_path = os.path.dirname(path)
+        with open(os.path.join(dir_path, "hazelcast.xml")) as f:
+            return f.read()
+
+    @classmethod
+    def configure_client(cls, config):
+        config["cluster_name"] = cls.cluster.id
+        return config
+
+    def setUp(self):
+        name = random_string()
+        self.client.create_vector_collection_config(name, [IndexConfig("vector", Metric.COSINE, 3)])
+        self.vector_collection = self.client.get_vector_collection(name).blocking()
+
+    def tearDown(self):
+        self.vector_collection.destroy()
+
+    def test_set(self):
+        doc = Document("v1", Vector("vector", Type.DENSE, [0.1, 0.2, 0.3]))
+        self.vector_collection.set("k1", doc)
+
+    def test_get(self):
```

Comment:
this can be renamed to get_set and the test above can be removed (test_set)

Reply:
I'd like to keep it as is.